src/main/resources/entity-types/shared/simple_language.json5

```diff
@@ -1,23 +0,0 @@
-{
-  id: 'a435d29f-ff11-4b7a-8a5f-3c5505452208',
-  name: 'simple_language',
-  private: true,
-  sources: [
-    {
-      alias: 'languages',
-      type: 'db',
-      target: 'drv_languages',
```

Comment:
This ET has been unusable since removal of materialized views. No other entities refer to it, either.